runtime/autoload/man.vim

```diff
@@ -50,7 +50,7 @@ function! man#open_page(count, count1, mods, ...) abort
     return
   endtry
   call s:push_tag()
-  let bufname = 'man://'.name.(empty(sect)?'':'('.sect.')')
+  let bufname = fnameescape('man://'.name.(empty(sect)?'':'('.sect.')'))
```

Comment:
better to call fnameescape() directly where it actually needs to be escaped. Or else rename this variable to `bufname_esc`